### PR TITLE
Add a new Question

### PR DESCRIPTION
**Pull request by dazzle-sah**
@Ebazhanov I have added a new question in Linux Section.
If you are merging this PR , then please give it the tag of Hacktoberfest-accepted, as I have contributed this as a part of Hacktoberfest 2022.

### CODE REVIEW

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
merge conflicts

File: html/html-quiz.md [hunk 1]
```diff
@@ -1930,9 +1930,16 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
...
 - [ ] <lb>
 - [x] <br>
 - [ ] <a>
+
+#### Q113. The HTML global attribute, "contenteditable" is used to:
+
+- [ ] Update content from the server
+- [ ] Specifies a context menu for an element. The menu appears when a user right-clicks on the element.
+- [x] Specify whether the content of an element should be editable or not
+- [ ] Return the position of the first found occurrence of content inside a string
```
Comment (auchynnikau) [inline html/html-quiz.md:1945]: Your changes are duplicate each other. Please, check your PR #4843 it has 3 the same files with the same changes. Please split it, it's a good practice to create new PR for each topic.

**Comment by Ebazhanov** [comment]
@dazzle-sah ping

File: linux/linux-assessment.md [hunk 2]
```diff
@@ -799,6 +799,14 @@ if [[ $FILE == $GLOB ]] ;then
 - [ ] rename
 - [ ] ren
 
+
+#### Q96. Which of these do NOT constitute as a default part of the email system of Red Hat Linux?
```
Comment (artemu78) [inline linux/linux-assessment.md:803]: How is this related to HTML? Looks like it is a good question, but not really fit for HTML quiz.

File: python/python-quiz.md [hunk 3]
```diff
@@ -2191,6 +2191,14 @@ print("programming".center())
 - [x] Guido van Rossum
 - [ ] Alan Turing
 
+
+#### Q159. Which collection is ordered, changeable, and allows duplicate members?
```
Comment (artemu78) [inline python/python-quiz.md:2195]: It brakes questions number sequence, may be it worth to add question as a last, and simply increment number.  
Or you could change numbers of all next questions after this one.

**Comment by Ebazhanov** [comment]
@dazzle-sah pls fix merge conflicts, thx

File: php/php-quiz.md [hunk 4]
```diff
@@ -1349,3 +1349,11 @@ echo $smurf->name;
 3           $path . $_FILES['image']['name']);
 4  )
 ```
+
+
+#### Q93. Which superglobal variable holds information about headers, paths, and script locations?
```
Comment (auchynnikau) [inline php/php-quiz.md:1354]: This question is already exist.

**Comment by Ebazhanov** [comment]
@dazzle-sah your changes was merged via https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pull/5084

**Pull request closed without merging**